src/style.css

```diff
@@ -1 +1,17 @@
-/* styles go here */
+fieldset {
+  margin-bottom: 20px;
+}
+
+.fieldsetItem {
+  margin-top: 10px;
+  margin-bottom: 0;
+  display: block;
+}
+
+.fieldsetItem:first-of-type {
```

Comment:
You can use `:not(:last-of-type`)/`:not(:first-of-type)`  to set margin

Reply:
Hello) I did all that you asked)